.github/workflows/ohri-backend.yml

```diff
@@ -148,3 +148,134 @@ jobs:
           cache-to: type=gha,mode=max
       - name: Save image digest
         run: echo "::set-output name=image::${{ steps.build-and-push.output.digest }}"
+
+  dev_server_status:
+    if: github.ref == 'refs/heads/dev-deploy' 
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Wait for 5 minutos - Let Images reflect on Docker Hub
```

Comment:
What is the guarantee of the publish being completed in 5mins? is there another way to verify this?